Assignment4/Hugoms-2001/task06.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+# Task 06: Modifying RDF(s)
+
+
+# In[2]:
+
+
+get_ipython().system('pip install rdflib')
+
+
+# In[3]:
+
+
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2023-2024/master/Assignment4/course_materials"
+
+
+# In[4]:
+
+
+# Read the RDF file as shown in class
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+
+# In[5]:
+
+
+# Create a new class named Researcher
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+
+# In[6]:
+
+
+# TASK 6.1: Create a new class named "University"
+g.add((ns.University, RDF.type, RDFS.Class))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+
+# In[7]:
+
+
+# TASK 6.2: Add "Researcher" as a subclass of "Person"
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+
+# In[8]:
+
+
+#TASK 6.3: Create a new individual of Researcher named "Jane Smith"
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+
+# In[9]:
+
+
+#TASK 6.4: Add to the individual JaneSmith the email address, fullName, given and family names
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, VCARD.hasEmail, Literal("jsmith@example.org")))
```

Comment:
VCARD does not have this property. Look in the VCARD vocabulary for the right one to use.